app/Http/Controllers/PetController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Controllers;
+use App\Models\Pet;
+use Illuminate\Http\Request;
+
+class PetController extends Controller {
+
+    // create action method to return pets view
+    public function show(){
+        return view (
+            'pets',
+            [
+                'pets' => self::getPet(),
+            ]);
+            return null;
+    }
+
+    // method to return an array of pets
+    public static function getPet() {
```

Comment:
this can be private scope since it's only called within the class definition. also if returning an array of pets I'd name it `getPets`